### PR TITLE
New tutorial : 2016-05-03-how-to-use-deskaas-lab

### PR DESCRIPTION
**Pull request by lrivallain**

### CODE REVIEW

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Prerequisites: Having a manager access on ovh.com website.
+
+### From RunAbove website
+
+To order : Visit [https://www.runabove.com/deskaas.xml](https://www.runabove.com/deskaas.xml) and clic on "Start Now !"
+
+You'll be redirected to the order wizard and will receive a mail with deskaas login and url.
+
+### From OVH.com manager
```
Comment (VincentCasse) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: s/manager/customer panel/g

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "How to subscribe to and use the DeskaaS lab"
+categories: Labs
```
Comment (VincentCasse) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: s/vdi/Labs/g

**Comment by VincentCasse** [comment]
Hi @lrivallain,

Two things : 
1/ Can you sign your commit please ? ```git commit -s --amend```
2/ Can you fix the categorie name and replace 'manager' by 'customer panel' please?

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md (hunk 1 above)
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: customer interface. Panel is more like a jury ;)

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+author: lrivallain
+lang: en
+---
+
+## Ordering process
```
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: "How to join the lab?", "Joining the lab"

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Prerequisites: Having a manager access on ovh.com website.
```
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: Beforehand, make sure to have an OVH.com account. If you don't, you may create one from https://www.ovh.com/manager/web/login/ and select "Create Account"

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+## Available offers
+
+Currently, only one offer is available :
+
+*   1 core + 2GB RAM + 10GB Storage + Windows OS
+
+![Lab](/kb/images/2016-05-03-how-to-use-deskaas-lab/offers.png)
+
+It's planned to quickly add more models with more storage, RAM or CPU cores.
```
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: We are planning to... Stay tuned!

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+
+## Quickstart
+
+### Web client access
+
+The delivery mail contains an URL https://*.deskaas.com to join the web client:
```
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: ...contains a URL of the form...

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+![soft_authentication](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_authentication.png)
+
+Then, the list of available desktop is displayed :
+
+![soft_list-desktops](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_list-desktops.png)
+
+And virtual desktop is ready to used:
+
+![soft_logged-in](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_logged-in.png)
```
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: May be interesting to include a useful links section here. Like, where to order, where to get more informations, where is the mailing list, where to get support, ....

**Comment by lrivallain** [comment]
Thanks for your comments. All taken into account in 4ec9aa8.
(also done some crappy (things|commits) when trying to sign first commit sorry)

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 8]
```diff
@@ -34,14 +34,14 @@ Currently, only one offer is available :
...
-It's planned to quickly add more models with more storage, RAM or CPU cores.
+We are planned to quickly add more models with more storage, RAM or CPU cores. Stay tuned!
 
 
 ## Quickstart
 
 ### Web client access
 
-The delivery mail contains an URL https://*.deskaas.com to join the web client:
+The delivery mail contains an URL in the form of https://*.deskaas.com to join the web client:
```
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: of the form, sorry :s

File: en/_posts/2016-05-03-how-to-use-deskaas-lab.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+![Lab](/kb/images/2016-05-03-how-to-use-deskaas-lab/offers.png)
+
+We are planned to quickly add more models with more storage, RAM or CPU cores. Stay tuned!
+
+
+## Quickstart
+
+### Web client access
+
+The delivery mail contains an URL in the form of https://*.deskaas.com to join the web client:
```
Comment (yadutaf) [inline en/_posts/2016-05-03-how-to-use-deskaas-lab.md]: of the form

**Comment by yadutaf** [comment]
Looks good. Let me know if you need help to rebase + squash your commits

**Comment by lrivallain** [comment]
rebase and squash commits done. Thanks @yadutaf !

**Comment by yadutaf** [comment]
LGTM. @VincentCasse good for you ?

**Comment by VincentCasse** [comment]
LGTM

**Pull request merged**